packages/@n8n/config/src/configs/generic.config.ts

```diff
@@ -0,0 +1,15 @@
+import { Config, Env } from '../decorators';
+
+@Config
+export class GenericConfig {
+	/** Default timezone for the n8n instance. Can be overridden on a per-workflow basis. */
+	@Env('GENERIC_TIMEZONE')
+	timezone: string = 'America/New_York';
+
+	@Env('N8N_RELEASE_TYPE')
```

Comment:
Do we really want to keep the `geenric` section? or should we start moving these to more relevant sections, like `release.type`, and `system.gracefulShutdown.timeout` ?

Reply:
Agree, eventually let's reorganize this and some of the others to make more sense. For now focusing on bringing them all into the new style as they are now, also minimizes errors when porting.